src/hooks/useToggle.tsx

```diff
@@ -0,0 +1,13 @@
+import { useState } from "react";
+
+function useToggle() {
+  const [state, setState] = useState(true) as any;
+
+  const toggle = () => {
+    setState(!state);
+  };
+
+  return [state, toggle];
+}
+
+export default useToggle;
```

Comment:
you don't seem to actually be using this anywhere? if not needed please delete

Reply:
this left here as a handy toggle hook, in case we needed to use it in other parts of the App, but will delete and update again when needed.